LoopKitUI/GlucoseUnitPublishing.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  GlucoseUnitPublishing.swift
+//  LoopKitUI
+//
+//  Created by Nathaniel Hamming on 2021-01-13.
+//  Copyright © 2021 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+import HealthKit
+
+public protocol PreferredGlucoseUnitObserver {
+    func preferredGlucoseUnitDidChange(to preferredGlucoseUnit: HKUnit)
+}
+
+public protocol PreferredGlucoseUnitPublisher {
```

Comment:
No clients are consuming this protocol, there are just classes declaring conformance to it, and I don't understand a need for this protocol.

Reply:
fair point.